File: Problem 327P1/0327_Problem_1.py
# Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
# LeetCode Benchmark: 141 ms

def maximumCount(nums: List[int]) -> int:
    return max(
        len([x for x in nums if x < 0]),
        len([x for x in nums if x > 0]))

File: Problem 327P1/0327_Problem_1.swift
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 114 ms

func maximumCount(_ nums: [Int]) -> Int {
    max(nums.filter({ $0 > 0 }).count, 
        nums.filter({ $0 < 0 }).count)
}
